common/ui/menubar.cpp

```diff
@@ -751,8 +751,12 @@ void menubar_init( const char * file )
 			j.Hotkey = -1;
 	auto infile = PHYSFSX_openReadBuffered(file);
 
-	if (!infile) return;
-		
+	if (!infile)
+	{
+		Warning("Couldn't find %s\n", file);
```

Comment:
Changed **all** occurrences of `Couldn't` to `Could not` (except the one in text.h, which reads from the hog file).